contrib/mupdf/template.py

```diff
@@ -0,0 +1,49 @@
+pkgname = "mupdf"
+pkgver = "1.23.9"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_args = [
+    "build=release",
+    "shared=yes",
+    "USE_SYSTEM_LIBS=yes",
+    "USE_SYSTEM_MUJS=yes",
+    "USE_SYSTEM_GLUT=yes",
```

Comment:
the forked version it uses is quite different so i'm not sure if this is good or not